test/contracts/CityDAOParcelDrop/CityDAOParcelDrop.spec.ts

```diff
@@ -0,0 +1,277 @@
+import { expect } from "chai";
+import { ethers, network } from "hardhat";
+import { MerkleTree } from 'merkletreejs';
+import merkle_leaves from './merkle-leaves.json';
+import keccak256 from 'keccak256';
```

Comment:
this isn't working probably because it's not added to package.json, but ethers provides its own keccak256 so you can use that. check the Roles constants file for an example

Reply:
That might be better to use ethers.keccak256 for consistency.